bot/exts/halloween/hacktoberstats.py

```diff
@@ -242,9 +246,13 @@ async def get_october_prs(github_username: str) -> List[dict]:
             f"&per_page={per_page}"
         )
 
-        headers = {"user-agent": "Discord Python Hacktoberbot"}
         async with aiohttp.ClientSession() as session:
-            async with session.get(query_url, headers=headers) as resp:
+            async with session.get(f"https://api.github.com/users/{github_username}", headers=REQUEST_HEADERS) as resp:
+                if resp.status == 404:
+                    logging.debug(f"No GitHub user found named '{github_username}'")
+                    return
```

Comment:
This seems to be an unnecessary extra call for every command usage; we already know if it's a user we can't read based on the response we get with the original request, we just need to avoid logging it as an error as it's an intended behaviour.

Reply:
Yeah, I actually thought about that right after I submitted the PR. I've unnecessarily added more complexity and an extra request per command invocation. Addressed in deee8cf!